rfhub2/cli/population.py

```diff
@@ -0,0 +1,175 @@
+# from pathlib import Path
+import robot.libraries
+from robot.errors import DataError
+from robot.libdocpkg import LibraryDocumentation
+from requests import session, post
+from rfhub2.config import APP_INTERFACE, APP_PORT, BASIC_AUTH_USER, BASIC_AUTH_PASSWORD
+import os
+import re
+
+RESOURCE_PATTERNS = {".robot", ".txt", ".tsv", ".resource"}
+ALL_PATTERNS = (RESOURCE_PATTERNS | {".xml", ".py"})
+PROTOCOL = 'http://'
```

Comment:
it shouldn't be hardcoded, should be part of user provided api url instead

Reply:
moved as a part of app-protocol

Reply:
Changed parameter to app-url, default value is `http://localhost:8000`